.github/workflows/nightly_test.yml

```diff
@@ -3,8 +3,8 @@ on:
   workflow_dispatch:
 
   schedule:
-    # Run every day at 0 minutes and 0 hours (midnight GMT)
-    - cron: "0 0 * * *"
+    # Run every day at 06:00 GMT (roughly 10pm PST)
+    - cron: "0 6 * * *"
```

Comment:
maybe we should move this a bit earlier so that it's not middle of the day for other folks? 7 or 8pm PT would still be "off hours" to a large extent